shinydashboard/lantern/ui.R

```diff
@@ -25,6 +25,7 @@ ui <- dashboardPage(
       menuItem("Values", icon = icon("table"), tabName = "values_tab", badgeLabel = "new", badgeColor = "green"),
```

Comment:
I think we can get rid of the new badge on this tab, since the Validations tab is the newest one!